packages/zeplin-extension-style-kit/declarations/backgroundColor.d.ts

```diff
@@ -8,7 +8,7 @@ declare class BackgroundColor {
 
     equals(other: BackgroundColor): boolean;
 
-    getValue(params: ColorParams, variables: VariableMap): string;
+    getValue(params: ColorParams, container: object, formatVariable: (colorObject: object) => string): string;
```

Comment:
This doesn't match the signature of the implementation. There are some other files having this issue too.
```suggestion
    getValue(params: ColorParams, formatVariable: (colorObject: object) => string): string;
```